tests/testHomework.js

```diff
@@ -327,7 +329,32 @@ describe('Homework assessment', function() {
         helperAttachFiles.checkNoAttachedFiles(locals);
     });
 
-    describe('6. instance_question: attach files setup', function() {
+    describe('6. assessment_instance: download deprecated text file from filesystem ', function() {
+        const filepath = 'testpath/';
+        const filename = 'test.zyx';
+
+        before('should insert deprecated file', async function() {
+            const content = 'This is the test text';
+            fs.mkdirSync(path.join(config.filesRoot, filepath));
```

Comment:
Are all these `fs` calls intentionally *Sync?

Reply:
```suggestion
            await fs.mkdir(path.join(config.filesRoot, filepath));
```

Reply:
They were, but do not have to be at all. I will change that now.

Reply:
![image](https://user-images.githubusercontent.com/7872295/94442639-0dca2200-0159-11eb-9f2d-c9f62fd9ca51.png)

I didn't know this, but we can return a promise in a slightly different way:

ie. fs.promises.mkdir('whatever/path')

Reply:
Fixed here: 54d589c15e5cd322325a04232bef3f4355c7cad1